### ISSUE TITLE
Add a "check zones" feature

### ISSUE THREAD

**Issue by mkonieczny9805** [brobertson/Lace2#103]
For long texts, it would be useful to have a feature that checks whether there are any pages that haven't been zoned

**Comment by brobertson** [comment]
Agreed! Perhaps those that have edited texts but no zoning?

**Comment by mkonieczny9805** [comment]
Yes, since usually only a portion of any given image set contains the target text to be edited.
